Userland/Libraries/LibWeb/HTML/StructuredSerialize.cpp

```diff
@@ -772,6 +791,31 @@ class Deserializer {
         m_position += 2;
         return value;
     }
+
+    static WebIDL::ExceptionOr<JS::NonnullGCPtr<Bindings::PlatformObject>> create_serialized_type(u32 name, JS::Realm& realm)
+    {
+        switch (static_cast<ValueTag>(name)) {
+        case ValueTag::BlobObject:
+            return FileAPI::Blob::create(realm);
+        default:
+            VERIFY_NOT_REACHED();
+        }
+    }
+
+    // FIXME: Consolidate this function with the similar is_interface_exposed_on_target_realm() used when transferring objects.
+    //        Also, the name parameter would be better off being the interface name (as a string) so that we don't need a switch statement.
+    static bool is_interface_exposed_on_target_realm(u32 name, JS::Realm& realm)
```

Comment:
Wait, why couldn't we serialize this as `ValueTag::Serializable` and actually add the Serializable's interface name to this, to be closer to what Transferrable does? Then we wouldn't need to expose the Enum in the header. I'm not sure adding the extra helper method here is worth the trouble when we could actually do it the same way as Transferrable :thonk: